package.json

```diff
@@ -62,6 +62,7 @@
     "babel-core": "^6.2.1",
     "babel-preset-es2015": "^6.1.18",
     "batch": "^0.5.1",
+    "fastly": "Financial-Times/fastly#v2.5.0",
```

Comment:
i recommend this for fastly deployments:- https://github.com/Financial-Times/fastly-tools

we don't really maintain this other one anymore i think.